### PR TITLE
documentation/api-products

### PR DESCRIPTION
**Pull request by ntngoc96**
documentation/api-products

### CODE REVIEW

File: docs/api/products-api.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - type: string
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 10
+        2. startWith: [a-z] || [A-Z]
+        3. oneOf: ['food','drinks']
+4. image
+    - type: string
+    - required: optional
+    - regexp: length: min: 1 - max: 1000
```
Comment (nmtoan251998) [inline docs/api/products-api.md]: **Param length**
What does length here means? image *size*?

File: docs/api/products-api.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+5. price
+    - type: double
```
Comment (nmtoan251998) [inline docs/api/products-api.md]: **Modifier format**
MongoDb does not support *double* type, do you mean *Number*?

File: docs/api/products-api.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    - required: true
+    - regexp: length: min: 3
+6. description 
+    - type: string
+    - required: optional
+    - regexp: length: min: 50 - max: 150
```
Comment (nmtoan251998) [inline docs/api/products-api.md]: **Param length**
I think we should delete the minimum length validation due to some conveniences I've just firgured out
min: 50 -> *undefined*

File: docs/api/products-api.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 100
+        2. startWith: [a-z] || [A-Z]
+3. type
+    - type: string
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 10
+        2. startWith: [a-z] || [A-Z]
```
Comment (nmtoan251998) [inline docs/api/products-api.md]: **RegEx**
I think /drink|food/ will definitely match the [a-zA-Z]

File: docs/api/products-api.md (hunk 1 above)
Comment (ntngoc96) [reply docs/api/products-api.md]: hmm... image's name bro!

File: docs/api/products-api.md (hunk 2 above)
Comment (ntngoc96) [reply docs/api/products-api.md]: omg I dunt remember :((

File: docs/api/products-api.md (hunk 3 above)
Comment (ntngoc96) [reply docs/api/products-api.md]: Changing to 10 character is better. Because products should have description.

**Pull request merged**